pkg/authorization/authz.go

```diff
@@ -163,3 +163,16 @@ func headers(header http.Header) map[string]string {
 	}
 	return v
 }
+
+// authorizationError represents an authorization deny error
+type authorizationError struct {
+	error
+}
+
+func (e authorizationError) HTTPErrorStatusCode() int {
```

Comment:
Should probably be documented, as it's an exported function;

    // HTTPErrorStatusCode returns a status code.